app/fissile.go

```diff
@@ -835,14 +901,22 @@ func (f *Fissile) GenerateKube(rolesManifestPath, outputDir, repository, registr
 		FissileVersion:  fissileVersion,
 		Opinions:        opinions,
 		Secrets:         refs,
+		CreateHelmChart: createHelmChart,
 	}
 
 	for _, role := range rolesManifest.Roles {
 		if role.IsDevRole() {
 			continue
 		}
+		if createHelmChart && role.Run.FlightStage == model.FlightStageManual {
```

Comment:
I think they probably should have their own charts. They are not really part of the deployment itself. Please file a new ticket for them.

Right now you can still run `make kube` and get the corresponding kube config file.